src/sdk/formatting/formatDateRange.ts

```diff
@@ -4,10 +4,10 @@ import { formatDate, DateFormatOptions } from './formatDate';
 
 export const formatDateRange = (start: ConvertibleDate, end: ConvertibleDate, options?: RangeFormatOptions & DateFormatOptions): string => {
 	options = options || {};
-	options.absolute = true;
+	options.allowRelative = false;
 
-	if (typeof(options.time) !== 'boolean') {
-		options.time = false;
+	if (typeof(options.displayTime) !== 'boolean') {
+		options.displayTime = false;
```

Comment:
is there a test that tests this piece?

Reply:
Sort of since we don't define it in these tests: https://github.com/portalarchitects/skysync-cli/blob/sf-2548/src/sdk/formatting/formatDateRange.test.ts#L12

Reply:
...i guess.... ;)